waiter/src/waiter/token.clj

```diff
@@ -201,9 +201,9 @@
     "List all tokens for a given user."
     [kv-store owner]
     (let [owner->owner-key (kv/fetch kv-store token-owners-key)]
-      (if-let [owner-key (owner->owner-key owner)]
+      (if-let [owner-key (get owner->owner-key owner)]
```

Comment:
Why was this change needed?

Reply:
Because `owner->owner-key` is `nil` in some cases (e.g. waiter just started up with an no tokens)